Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -264,6 +264,18 @@ class BrowserViewController: UIViewController {
             if result == .walletCreated {
                 self.rewards.ledger.setupDeviceCheckEnrollment(client) { }
             }
+            
+            let isRewardsPanelShowing = (self.presentedViewController as? PopoverController)?
+                .contentController is RewardsPanelController
```

Comment:
This is supposed to happen when the user enables rewards via the NTP notification correct? In that case, it should probably be part of the above check (`result == .walletCreated`)

Also we technically already have code that checks if an ad can show here: https://github.com/brave/brave-ios/blob/5eab26a129be31811365cf74961e8b08fe2658cf/BraveRewardsUI/Ads/AdsNotificationHandler.swift#L105-L127

Perhaps we could reuse that instead somehow?